### ISSUE TITLE
about discriminator output activation

### ISSUE THREAD

**Issue by learnxbgl** [manicman1999/Keras-BiGAN#1]
hi, i noticed that on the discriminator output not applied nay activation, is that ok?

**Comment by manicman1999** [comment]
Yes! The hinge loss function (the one used in this implementation) allows for any values from the Discriminator, and does not limit the output to a certain range. This helps to prevent vanishing gradients.
